### ISSUE TITLE
Infinitely long growth is allowed

### ISSUE THREAD

**Issue by molpopgen** [grahamgower/demes#26]
I discovered that this is allowed.  The population has exponential growth set by default by also a start time of `inf` by default.

```yaml
description: I believe this is a bad model.  Ancestral is described to grow from infinitely back in the past until 220e3 in the past.
doi: None
time_units: years
generation_time: 25
demes:
  ancestral:
    description: Equilibrium/root population
    end_time: 220e3
    initial_size: 7300
    final_size: 10000
```

**Comment by apragsdale** [comment]
Right. We'd need to impose that the first epoch (or if it's just one epoch, like here) is a constant size function (initial = final size) if the deme has a start time of inf.

**Comment by apragsdale** [comment]
Closed by #27
